tests/test_wasm_intrinsics_simd.c

```diff
@@ -1275,16 +1255,10 @@ int EMSCRIPTEN_KEEPALIVE __attribute__((__optnone__)) main(int argc, char** argv
     v128_xor((v128_t)i32x4(0, 0, -1, -1), (v128_t)i32x4(0, -1, 0, -1)),
     i32x4(0, -1, -1, 0)
   );
-
-#ifdef __wasm_unimplemented_simd128__
-
   expect_vec(
     v128_andnot((v128_t)i32x4(0, 0, -1, -1), (v128_t)i32x4(0, -1, 0, -1)),
-    i32x4(0, -1, -1, 0)
+    i32x4(0, 0, -1, 0)
```

Comment:
huh... what happened here?

Reply:
Since this test was behind the unimplemented flag, it was never run and it turns out it was just wrong.